app/views/layouts/_footer.html.erb

```diff
@@ -10,7 +10,7 @@
           <div class="row">
             <% presenter.nav_links.each do |nav_link| %>
               <div class="col-md-4">
-                <%= link_to nav_link[:title], nav_link[:url], class: 'footer-column__link' %>
+                <%= link_to nav_link[:title], nav_link[:url], { class: 'footer-column__link' }.merge(nav_link[:target_blank] || {} ) %>
```

Comment:
and in here

```erb
<%= link_to nav_link[:title], nav_link[:url], class: 'footer-column__link', **nav_link[:attributes] %>
```

not entirely sure it'll work, but worth a try

Reply:
This will crash for `nav_links` which don't have the `:attributes` key.